admin/inc/class_form.php

```diff
@@ -816,20 +816,11 @@ function generate_date_select($name, $day=0,$month=0,$year=0)
 	{
 		global $lang;
 
-		$months = array(
-			1 => $lang->january,
-			2 => $lang->february,
-			3 => $lang->march,
-			4 => $lang->april,
-			5 => $lang->may,
-			6 => $lang->june,
-			7 => $lang->july,
-			8 => $lang->august,
-			9 => $lang->september,
-			10 => $lang->october,
-			11 => $lang->november,
-			12 => $lang->december,
-		);
+		$months = array();
+		for($m = 1; $m <= 12; $m++)
+		{
+			$months[$m] = $lang->{'month_'.$m};
```

Comment:
These language changes will need big bold comments in the release notes to make it obvious that language string names have changed. Ping @dvz for awareness.